docs/docusaurus/docs/core/_create_expectations/expectation_suites/manage_expectation_suites.md

```diff
@@ -24,9 +24,16 @@ An Expectation Suite contains a group of Expectations that describe the same set
 
 ## Create an Expectation Suite
 
-<Tabs>
+<Tabs 
+   queryString="procedure"
+   defaultValue="instructions"
+   values={[
```

Comment:
Are you required to pass in `values` if you pass in the other parameters? Feels tedious!

Reply:
You aren't, but I've been plugging these in using a pycharm live template so it actually is quite quick: I just select the content I want to put in the "Instructions" tab, press a few buttons and whoosh! Instant tab group with header and Instructions/Sample Code tabs.  Add the code block for the full example to the Sample Code tab and done.